pkg/loader/compose/v3.go

```diff
@@ -654,20 +655,22 @@ func handleV3Volume(komposeObject *kobject.KomposeObject, volumes *map[string]ty
 	}
 }
 
-func getV3VolumeLabels(name string, volumes *map[string]types.VolumeConfig) (string, string) {
-	size, selector := "", ""
+func getV3VolumeLabels(name string, volumes *map[string]types.VolumeConfig) (string, string, string) {
+	size, selector, storageClassName := "", "", ""
 
 	if volume, ok := (*volumes)[name]; ok {
 		for key, value := range volume.Labels {
 			if key == "kompose.volume.size" {
 				size = value
 			} else if key == "kompose.volume.selector" {
 				selector = value
+			} else if key == "kompose.volume.storageClassName" {
+				storageClassName = value
```

Comment:
Please read the user guide and compare to other kompose labels, I think the labels do not use CamelCase

Reply:
updated